splunk_add_on_ucc_framework/ucc_ui_lib/src/main/webapp/components/TextComponent.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Text from '@splunk/react-ui/Text';
+
+class TextComponent extends Component {
+    constructor(props) {
+        super(props);
+    }
+
+    handleChange = (e) => {
+        this.props.handleChange(this.props.id, e.target.value);
+    };
+
+    render() {
+        return (
+            <Text
+                inline
+                placeholder={this.props.controlOptions.placeholder}
+                className={this.props.field}
+                disabled={this.props.controlOptions.display === false ? true : false}
```

Comment:
I think, `this.props.controlOptions.display` is used in form for to hide the component. That would be handle by ControlWarpper.
For, `disabled` you will receive `this.props.disabled `.

Reply:
Updated